sdcm/sct_config.py

```diff
@@ -499,6 +499,9 @@ class SCTConfiguration(dict):
              In case of random option - Scylla will start with different (random) shards on every node of the cluster
              """),
 
+        dict(name="nodes_smp", env="SCT_NODES_SMP", type=list,
```

Comment:
```suggestion
        dict(name="nodes_smp", env="SCT_NODES_SMP", type=str_or_list_or_eval,
```

that way one can override it from environment variables if needed

Reply:
I wonder maybe we could have a bit more descriptive name for the configuration option
`fixed_smp_per_db_node` 
or
`smp_per_db_node_mapping`